web/src/main/java/cz/muni/fi/pa165/security/CustomAuthenticationProvider.java

```diff
@@ -39,8 +39,7 @@ public Authentication authenticate(Authentication authentication) throws Authent
             PersonDTO person = personFacade.findUserByLogin(username);
 
             List<GrantedAuthority> grantedAuths = new ArrayList<>();
-            grantedAuths.add(new SimpleGrantedAuthority("ROLE_" + person.getRole().toString()));
-
+            grantedAuths.add(SimpleGrantedAuthorityFactory.createAuthority(person.getRole().toString()));
```

Comment:
Funguje to bez toho ROLE_? Pretoze pokial viem to ROLE je nejaka poziadavka v springu a myslim ze mi to bez toho hadzalo chybu. Ja to potom skusim ked pridem domov, teraz som v praci.